### PR TITLE
adding the basic Proof of Concept code that the system can be based o…

### PR DESCRIPTION
**Pull request by BrianGenisio**
…ff of

### CODE REVIEW

File: heroku-server-example.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+var app = require('express')();
+var server = require('http').createServer(app);
+var io = require('socket.io')(server);
+
+app.set('port', (process.env.PORT || 3000));
+
+app.get('/ece926d8c0356205276a45266d361161', function(request, response) {
+    response.send("OK");
+    io.emit('action');
```
Comment (MiguelCastillo) [inline heroku-server-example.js:9]: Who is publishing `count` and who is listening for `action`?

Comment (BrianGenisio) [inline heroku-server-example.js:9]: Good point.  I had these files lying around, but clearly, they weren't in sync.  This is the basic idea, though.  Consider this code untested.

I updated the client to be more clear.

File: node-client-example.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 var socket = require('socket.io-client')('https://young-fjord-53100.herokuapp.com');
 
-socket.on('count', function(count){
-    console.log('Do something: ', count);
-});
\ No newline at end of file
+socket.on('action', function(data){
```
Comment (MiguelCastillo) [inline node-client-example.js:3]: Nice!

**Comment by MiguelCastillo** [comment]
@daniellecloss I merged this PR.  It might worse collecting everyones github usernames that are interested in this project so that we can rope them in via mentions.

**Pull request merged**